### DISCUSSION TITLE
How long will github reply a ticket?

### DISCUSSION THREAD

**Discussion by hellodword** [community/community#21647] [New to GitHub]
<p>My ticket was created 11 days ago, and a ticket of my friend was created 21 days gao, no response, nothing.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Ticket reply times depend on quite a few variables - we have different speciality support squads where queues can vary substantially.</p>
<p>There are some tickets that can be replied to very quickly with a link to our documentation, and some that need to be reviewed by our legal or security teams which obviously can take quite a lot longer.</p>
<p>I can confirm both those tickets are in the correct queue, and will be responded to in order.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I can understand the confusion, but suspended accounts aren’t “account access” (that’s people whose accounts are not restricted but are unable to login for any number of reasons). If you can see that your account is suspended, you have - to some extent - accessed your account.</p>
<p>Account restrictions, such as being flagged or suspended, will require reviews from different teams - security, policy, engineering, etc - depending on why they were restricted. I understand that it can be distressing while you wait for the review to be completed.</p>
<p>I trust you can help your friend access any personal supports they may need during this wait.</p>
